convert2rhel/actions/__init__.py

```diff
@@ -540,17 +603,23 @@ def run_actions():
     results = system_checks.run()
 
     # Format results as a dictionary:
-    # {"$Action_id": {"status": int,
-    #                 "error_id": "$error_id",
-    #                 "message": "" or "$message"},
+    # {"$Action_id": {
+    #  "messages" : ["level": int,
+    #                "id": "$id",
+    #                "message": "" or "$message"}],
+    #  "result" : {
+    #      "level": int,
+    #      "id": "$id",
+    #      "message": "" or "$message"},
```

Comment:
```suggestion
    # Format results as a dictionary:
    # {
    #     "$Action_id": {
    #         "messages": [
    #             "level": int,
    #             "id": "$id",
    #             "message": "" or "$message"
    #         ]
    #     },
    #     "result": {
    #         "level": int,
    #         "id": "$id",
    #         "message": "" or "$message"
    #     },
    # }
```

Reply:
There was an error in the JSON were the order of closing the brackets was inverse. 

I updated that and formatted the JSON to look better in the comment.